lang/en.json

```diff
@@ -1306,6 +1306,7 @@
 "DND5E.DeathSaveRoll": "Roll a Death Saving Throw",
 "DND5E.DeathSavingThrow": "Death Saving Throw",
 "DND5E.DeathSaveUnnecessary": "You do not need to roll death saves because you have a positive number of hit points or have already reached 3 successes or failures.",
+"DND5E.DeductCurrencyError": "Failed to deduct {amount}{denomination} from {name}. Insufficient funds.",
```

Comment:
If we move `deduceCurrency` into `CurrencyManager`, then this should probably also move to `DND5E.CurrencyManager.Warning.InsufficientFunds`.